api-reference/field-component.md

```diff
@@ -56,9 +56,26 @@ const FieldExample = () => (
 )
 ```
 
-| Prop Name | Description |
-| :--- | :--- |
-| `children` | A single component. |
+## Props
+
+<table>
+  <thead>
+    <tr>
+      <th style="text-align:left">Prop Name</th>
+      <th style="text-align:left">Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td style="text-align:left"><code>children</code>
+      </td>
+      <td style="text-align:left">
+        <p>Node</p>
+        <p>A single component.</p>
```

Comment:
The second one of these should be wrapped in `<em>`.

Reply:
847b9e06cb75bdce733e3216442794e03167b340